pkg/webui/account/views/update-password/index.js

```diff
@@ -28,25 +27,26 @@ import Form from '@ttn-lw/components/form'
 import Input from '@ttn-lw/components/input'
 import SubmitButton from '@ttn-lw/components/submit-button'
 import Checkbox from '@ttn-lw/components/checkbox'
-import Spinner from '@ttn-lw/components/spinner'
 
 import Message from '@ttn-lw/lib/components/message'
 import IntlHelmet from '@ttn-lw/lib/components/intl-helmet'
 
-import style from '@account/views/create-account/create-account.styl'
+import style from '@account/views/front/front.styl'
```

Comment:
These styles belong to `front`, not to `update-password`

Reply:
I use the `front/front.styl` styles for all front (unauthenticated) views to avoid repetition.